groups/groups.yaml

```diff
@@ -446,6 +447,18 @@ groups:
       - stefan.schimanski@gmail.com
       - thockin@google.com
 
+  - email-id: k8s-infra-rbac-release-triageparty@kubernetes.io
```

Comment:
Do we want `release-triageparty` or `triageparty-release` (given that we know that more groups may want to leverage Triage Party)?

Reply:
`triageparty-release` is more convenient for the possible scenario you described. Also naming stuff is hard :-).

@dims WDYT ?